app/assets/stylesheets/pages/discussion.scss

```diff
@@ -46,7 +46,6 @@
 
 .discussion-container {
   display: flex;
-  align-items: center;
```

Comment:
@gracesnoh is this alright? If I keep either align-items or the display flex, the discussions index list doesn't render underneath the menu bar (where the tags are)